.appcast.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0"
+     xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle"
+     xmlns:dc="http://purl.org/dc/elements/1.1/">
+  <channel>
+    <title>Sketch Favicon</title>
+    <link>
+      https://raw.githubusercontent.com/woofwoofinc/sketch-favicon/master/.appcast.xml
```

Comment:
This will always serve it as `text/plain`. You may want to consider using https://rawgit.com/ which can adopt appropriate mimetypes.

Reply:
From the rawgit production terms: "Files are cached permanently based on the URL." Pity. :( The contents of this file have to change if the user is to get the updated appcast.

Reply:
https://github.com/woofwoofinc/sketch-favicon/issues/27